controllers/storagecluster/external_resources.go

```diff
@@ -176,10 +176,11 @@ func (r *StorageClusterReconciler) newExternalCephObjectStoreInstances(
 		return nil, err
 	}
 	// enable bucket healthcheck
+	time60s, _ := time.ParseDuration("60s")
```

Comment:
You don't need to parse time anymore, just use `1 * time.Minute`

Reply:
Thanks, fixed (yet to push). Actually `time.Minute` is sufficient.